File: algorithms/SearchAndSort/SequentialSearch/sequentialSearch.go
package main

import (
	"fmt"
)

func sequentialSearch(myList []int, item int) bool {
	found := false
	pos := 0
	for pos < len(myList) && !found {
		if myList[pos] == item {
			found = true
		} else {
			pos++
		}
	}
	return found
}

func main() {
	myList := []int{1, 2, 3, 4}
	fmt.Println(sequentialSearch(myList, 4))
}

File: algorithms/SearchAndSort/SequentialSearch/sequentialSearch.js
function sequentialSearch(list, item) {
    var found = false
    var pos = 0
    while (pos < list.length && !found) {
        if (list[pos] == item) {
            found = true
        } else {
            pos++
        }
    }
    return found
}

console.log(sequentialSearch([1, 2, 3, 4], 4))